src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -107,6 +103,12 @@ const styles = StyleSheet.create({
   ml8: {
     marginLeft: 8
   },
+  marginNonContinuous: {
+    marginLeft: 4
+  },
+  marginContinuous: {
+    marginLeft: 5
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New margin styles (`marginNonContinuous` and `marginContinuous`) have been added. These are applied conditionally based on the `isContinuous` flag in the renderAvatar function. This seems to be a good approach to handle different conditions for continuous and non-continuous messages.
```diff
+  marginNonContinuous: {
+    marginLeft: 4
+  },
+  marginContinuous: {
+    marginLeft: 5
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->